src/cultionet/scripts/args.yml

```diff
@@ -85,7 +85,7 @@ shared_create:
     long: res
     help: The cell resolution
     kwargs:
-      default: 10.0
+      default: !!null

```

Comment:
If the default here is unchanged by the user, the cell resolution will be taken directly from the image.